app/layouts/DashboardLayout/DashboardLayout.presenter/DashboardLayout.presenter.tsx

```diff
@@ -3,11 +3,13 @@ import type { FC } from 'react';
 
 import { Box, makeStyles, useMediaQuery } from '@material-ui/core';
 import { ipcRenderer } from 'electron';
-
 // import { TIME_FOR_INACTIVITY, TIME_FOR_REACTION } from '../../../constants/app';
```

Comment:
delete comments

Reply:
cleaning up the view / view controllers is a nearly bottomless pit of effort, and we're bumping that kind of thing off to a future refactor where we simply rewrite the views from the ground up